### PR TITLE
add lambdas for session insights digests

### LINKED ISSUE #5721: Build a digest for "session insights" (with and w/o AI insights)
Session Insights email w/o "AI enhanced" session insights
Sessions insights email w/ "AI enhanced" session insights

### PR DESCRIPTION
**Pull request by mayberryzane**
## Summary
- closes #5721 
- creates 3 new lambdas:
  - `getSessionInsightsData` is a Go lambda, retrieves metadata about the top 3 "interesting" sessions for the project
  - `session-insights-email` is a Node.js lambda, renders an HTML email using react-email
  - `sendSessionInsightsEmails` is a Go lambda, sends the HTML email to all non-unsubscribed users in the project

## How did you test this change?
- deployed lambdas + tested in a step function: https://us-east-2.console.aws.amazon.com/states/home?region=us-east-2#/statemachines/view/arn:aws:states:us-east-2:173971919437:stateMachine:TestSessionInsights

## Are there any deployment considerations?
- will create a step function to chain these lambda executions

### CODE REVIEW

File: packages/react-email-templates/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+	"name": "emails",
+	"version": "0.0.18",
+	"private": true,
+	"workspaces": [
+		".react-email"
```
Comment (Vadman97) [inline packages/react-email-templates/package.json:6]: what's this for?

File: packages/react-email-templates/package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	],
+	"scripts": {
+		"build": "tsc --build ./",
```
Comment (Vadman97) [inline packages/react-email-templates/package.json:9]: for consistency with the `render` lambda, could we use `tsup` here too?
also do the dependencies get zipped correctly? i found that deps get hoisted up to the top level node_modules unless you add the following
```
	"installConfig": {
		"hoistingLimits": "dependencies"
	},
```

**Review by Vadman97 — APPROVED** [review]

File: packages/react-email-templates/package.json (hunk 1 above)
Comment (mayberryzane) [reply packages/react-email-templates/package.json]: this was how it was after the automatic setup https://react.email/docs/getting-started/automatic-setup - there's a .react-email subfolder but everything seems to work ok even if I delete this :)

File: packages/react-email-templates/package.json (hunk 2 above)
Comment (mayberryzane) [reply packages/react-email-templates/package.json]: Earlier I was testing by just building within the `react-email-templates` folder, added this config!

**Pull request merged**